shacl12-core/index.html

```diff
@@ -1053,58 +1210,124 @@ <h3>SHACL Example</h3>
 				"@id": "sh:ClosedConstraintComponent"
 			},
 			"sh:sourceShape": {
-				"@id": "sh:PersonShape"
+				"@id": "ex:PersonShape"
 			},
 			"sh:value": {
 				"@type": "xsd:date",
 				"@value": "1971-07-07"
-			}
+			},
+			"sh:resultMessage": "Predicate is not allowed (closed shape)"
 		}
 	]
 }</pre>
 						</div>
 					</div>
 				</aside>
+				<p><b>How to read the report:</b></p>
+				<ul>
+					<li>
+						<code>sh:ValidationReport</code> is the overall report, with <code>sh:conforms false</code> meaning at least one rule failed.
```

Comment:
```suggestion
						<code>sh:ValidationReport</code> is the overall report, with <code>sh:conforms false</code> meaning that there was at least one violation.
```

The suggested change should better match the logic of `sh:conforms` #453 without overwhelming people with the full logic in the getting started section.